packages/data-stores/src/domain-suggestions/types.ts

```diff
@@ -100,22 +121,15 @@ export interface DomainSuggestion {
 	 */
 	match_reasons?: readonly string[];
 
-	/**
-	 * Rendered cost with currency
-	 *
-	 * @example "€15.00"
-	 */
-	cost: string;
-
 	/**
 	 * The product ID
 	 */
-	product_id: number;
+	product_id?: number;
 
 	/**
 	 * The product slug
 	 */
-	product_slug: string;
+	product_slug?: string;
```

Comment:
Switched a bunch of properties to optional because the free suggestion returned by the API doesn't include them.

Reply:
Why are `relevance`, `supports_privacy`, `vendor`, `product_id` & `product_slug` not included in the free suggestion and therefore optional?